pkg/job/job.go

```diff
@@ -79,8 +79,17 @@ func (j *Job) Pause() {
 // IsCancelled returns whether the job has been cancelled
 func (j *Job) IsCancelled() bool {
 	select {
-	case _, ok := <-j.CancelCh:
-		return !ok
+	case <-j.CancelCh:
+		return true
+	default:
+		return false
+	}
+}
+
+func (j *Job) IsPaused() bool {
+	select {
+	case <-j.PauseCh:
+		return true
 	default:
 		return false
 	}
```

Comment:
In the enabled unit-test it is required to check if a job was paused. Therefore adding this method (it will be used below).